CodingTracker.Speedierone/CodingTracker.Speedierone/Helpers.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.ComponentModel.Design;
```

Comment:
Please remove unused using.

![image](https://user-images.githubusercontent.com/108711476/231962076-a57076b3-19a0-4023-b11d-f34abccd65d9.png)

Reply:
Heya, 
This isnt finished yet I think this one is going to take me longer. Ran out of time yesterday to do any more so just pushed it as a backup. Hope thats okay :)